Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
This is required for correct feature selection. With resolver v1, if we had a dev-dependency on runtime with the `testing-utils`, it would also select the feature for non-dev builds. Version 2 correctly does the following:

> Features enabled on https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies will not be unified when those same dependencies are used as a normal dependency, unless those dev-dependencies are currently being built.